tensorboard/plugins/histogram/summary_test.py

```diff
@@ -119,6 +119,16 @@ def test_when_bucket_count_not_statically_known(self):
         buckets = tensor_util.make_ndarray(pb.value[0].tensor)
         self.assertEqual(buckets.shape, (bucket_count, 3))
 
+    def test_with_large_counts(self):
+        # Check for overflow with large count (2^24) of data.
+        large_count = 2 ** 24
+        data = [0] + [1] * large_count
+        pb = self.histogram("large_count", data=data, buckets=2)
+        buckets = tensor_util.make_ndarray(pb.value[0].tensor)
+        np.testing.assert_allclose(
```

Comment:
We shouldn't be using allclose() here, because that allows some error in the values - specifically, it allows a relative difference of `rtol=1e-07` by default.  And the difference we want to measure could be as small as between 16777216 and 16777217, which is an error of only 1/16777216 = 5.96e-8, so it's small enough that `allclose()` says this is close enough to count as equal.

I'd recommend just doing `self.assertEqual(buckets[1][2], large_count)`, since we expect this result to be exactly correct.

Reply:
Done.